src/app/index.js

```diff
@@ -9,7 +9,8 @@ import Module from '../module';
 import Route from '../route';
 import title from '../_/title';
 
-export default define('sk-app', class extends Component {
+export default define(class extends Component {
+  static get is(){ return 'sk-app' }
```

Comment:
yeah I to the former all the time :D as we had discussion previously getter is immutable, property not.